### PR TITLE
COMPLETED / Landing page items positioning

### PR DESCRIPTION
**Pull request by belminmostic**
--------------------------------------------------------------------
MAIN
* Changed js and scss files structure for home
--------------------------------------------------------------------
OTHER
* Refactored footer
--------------------------------------------------------------------

### CODE REVIEW

File: src/Home/OurAim/_ouraim.scss [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+  .our-aim-container--picture{
+    display:flex;
+    justify-content: space-between;
+    object-fit: contain;
+    padding-bottom: 79px;
+    // img{
```
Comment (kkuna) [inline src/Home/OurAim/_ouraim.scss]: Remove left over comments 📟

File: src/Home/StartupIdea/StartupIdea.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import pic1 from '../../Images/scenes-03-2.png';
+
+const StartupIdea = () => (
+  <div className="startup-idea">
+    <div className="container">
+      <h1>Have a startup idea? Tell us more</h1>
+      <p>Feel free to ask for tips and guidance to validate and skyrocket your startup idea into reality</p>
+      <button className="btn startup-idea-button">LET US KNOW</button>
+      <img src={pic1} alt="pic" />
```
Comment (kkuna) [inline src/Home/StartupIdea/StartupIdea.js]:
```suggestion
      <img src={pic1} alt="ActualAltTag" />
```

Add more descriptive alt's, something like Startup Idea Photo etc

File: src/Home/Workflow/_workflow.scss [hunk 3]
```diff
@@ -0,0 +1,20 @@
+// Workflow
+.workflow{
+  padding-top: 130px;
+}
+.workflow-column{
+  display:flex;
+  flex-direction: column;
+  align-items: flex-start;
+  justify-content: flex-end;
+  // align-items: flex-end;
```
Comment (kkuna) [inline src/Home/Workflow/_workflow.scss]: Same as above, remove unused code 😄

File: src/Home/_home.scss [hunk 4]
```diff
@@ -1,4 +1,14 @@
 //index
+.h1{
```
Comment (kkuna) [inline src/Home/_home.scss]:
```suggestion
h1 {
```

`.h1` implies that the class name is h1, which it is not and should **NEVER** be

File: src/Home/_home.scss [hunk 5]
```diff
@@ -23,240 +33,27 @@ h2{
...
-  font-size: 10px;
-  background-color: #ffce45;
-  font-weight: bold;
-  color: #27272e;
-  letter-spacing: 1px;
-  text-align: center;
+  flex-wrap: wrap;
 }
-.diversity-row{
+.d-flex{
```
Comment (kkuna) [inline src/Home/_home.scss]: oh no! 
`.d-flex` is a bootstrap class, please do not override it, if you need to do something else except `display: flex`, either create a new class or use some other [bootstrap helpers](https://getbootstrap.com/docs/4.0/utilities/flex/)

File: src/Home/StartupIdea/StartupIdea.js (hunk 2 above)
Comment (tarikmehic) [reply src/Home/StartupIdea/StartupIdea.js]: Please use more declarative alt's throughout the website, it is very important for the SEO, as this is info website. 

https://www.mainstreetroi.com/alt-text-for-images-do-you-really-need-it/

**Pull request merged**